core/src/com/unciv/logic/BackwardCompatibility.kt

```diff
@@ -156,9 +156,8 @@ object BackwardCompatibility {
     }
 
     /** Move max XP from barbarians to new home */
-    @Suppress("DEPRECATION")
-    fun ModOptions.updateDeprecations() {
-    }
+    @Suppress("DEPRECATION", "EmptyFunctionBlock")
+    fun ModOptions.updateDeprecations() { }
```

Comment:
Why exactly are we calling this function?

Reply:
This is so if there's stuff to deprecate the devs can add the stages here. Currently there are none, but this is legit